testing/citest/tests/aws_smoke_test.py

```diff
@@ -245,17 +247,20 @@ def upsert_load_balancer(self, use_vpc):
      .call_method(
          self.elb_client.describe_load_balancers,
          LoadBalancerNames=[load_balancer_name])
-     .contains_path_match(
-         'LoadBalancerDescriptions',
-         {'HealthCheck': jp.DICT_MATCHES({
-             key: jp.EQUIVALENT(value) for key, value in health_check.items()}),
-          'AvailabilityZones': jp.LIST_SIMILAR(avail_zones),
-          'ListenerDescriptions/Listener':
-              jp.DICT_MATCHES(
-                  {key: jp.NUM_EQ(value)
-                   for key, value in listener['Listener'].items()})
-         })
-    )
+     .expect_value_list_contains(
```

Comment:
This is OK for now, but seems like we can provide a cleaner helper function when we're trying to match against a single payload object. I think the end goal for these should be to have them be expressive and clean to read and understand what's being validated.